package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fs-extra",
-  "version": "0.30.0",
+  "version": "0.30.1",
```

Comment:
sorry about that, I'll create a different branch and only cherry pick the first commit. Our automated build kept getting the original 0.30.0 version, so I updated it to work around it. But overlooked that it will automatically become part of my PR.